pkg/didcomm/protocol/didexchange/states.go

```diff
@@ -275,6 +276,28 @@ func (s *completed) ExecuteInbound(msg *stateMachineMsg, thid string, ctx *conte
 	case AckMsgType:
 		action := func() error { return nil }
 		return msg.connRecord, &noOp{}, action, nil
+	case CompleteMsgType:
+		complete := &Complete{}
+
+		err := msg.Decode(complete)
+		if err != nil {
+			return nil, nil, nil, fmt.Errorf("JSON unmarshalling of complete: %w", err)
+		}
+
+		action := func() error { return nil }
+
+		if msg.connRecord == nil {
+			return nil, &noOp{}, action, nil
+		}
+
+		connRec := *msg.connRecord
+
+		// Interop: aca-py did:sov DIDs are often sent without a did:sov: DID prefix, add the prefix here to handle it
```

Comment:
@Moopli Have you verified that AcaPy hasn't already fixed this? Is there an open issue?

Reply:
removed this specific code as I'm not sure this bit will actually be necessary, but yes, the sov stuff is still an issue that needs to be handled elsewhere.

Reply:
@Moopli I had thought that AcaPy might have introduced their own build flag to not emit sov. Have you double checked?

Reply:
Well, they don't have a startup flag for it, and I can't find anything in their didexchange manager or did doc implementation that would suggest the existence of a toggle, so I suspect not. I'll do some more digging if you want me to really really confirm though ;)

Reply:
@troyronda looks like we still need to accommodate sov "peer" for now